README.md

```diff
@@ -27,6 +27,36 @@ Installation instructions for the CLI
 
 This repository does not need any specific configuration.
 
+
+### Docker
```

Comment:
I am not sure if the build instructions should be in the Readme. Only `docker run --rm zonemtaster/cli zonemaster.net` is relevant for someone who wants to use it

Reply:
I agree. I will propose updates in a new PR when this has been merged.